Update documentation (#310)
Clinical-Genomics/BALSAMIC: docs/git_etiquette.rst

=============
Git Etiquette
=============

> It is recommended to loosely follow a system to standardize the commit messages. Following up from commit messages discussed on https://github.com/Clinical-Genomics/development/pull/97 , the format below is recommended for commit messages:

Reviewer: ```suggestion
It is recommended to follow a system to standardize the commit messages loosely. Following up from commit messages discussed on https://github.com/Clinical-Genomics/development/pull/97 , the format below is recommended for commit messages:
```

**Subject line**
^^^^^^^^^^^^^^^^^^^^

Subject can include one of the following:

#. **feat**\ : Introducing a new features. This includes but not limited to workflows, SnakeMake rule, cli, and plugins. In other words, anything that is new and fundamental change will also go here. Enhancements and optimizations will go into refactor.
#. **fix**\ : This is essentially a patch. Included but not limited to: bug fixes, hotfixes, and any patch to address a known issue.

> #. **doc**\ : Any changes to the documentations are part of doc subject line, included but no limited to: docstrings, cli-help, readme, tutorial, documentation, CHANGELOG, and addition of ipython/jupyter notebook in form of tutorial.

Reviewer: ```suggestion
#. **doc**\ : Any changes to the documentation are part of doc subject line, included but not limited to docstrings, cli-help, readme, tutorial, documentation, CHANGELOG, and addition of ipython/jupyter notebook in the form of tutorial.
```
